bookshelf-standard/3-binary-data/pom.xml

```diff
@@ -35,9 +35,10 @@ Copyright 2016 Google Inc.
     <bookshelf.storageType>datastore</bookshelf.storageType>   <!-- datastore or cloudsql -->
 
     <sql.dbName>bookshelf</sql.dbName>                        <!-- A reasonable default -->
-<!-- Instance Connection Name - project:region:dbName -->
-<!-- -Dsql.instanceName=localhost to use a local MySQL server -->
-    <sql.instanceName>DATABASE-connectionName-HERE</sql.instanceName> <!-- See `gcloud sql instances describe [instanceName]` -->
+    <!-- Instance Connection Name - project:region:dbName -->
+    <!-- -Dsql.instanceName=localhost to use a local MySQL server -->
+    <sql.instanceName>DATABASE-connectionName-HERE
```

Comment:
Same as above. It would also be nice if it was `YOUR_INSTANCE_CONNECTION_NAME` instead of `DATABASE-connectionName-HERE`.